packages/sessions-sdk-react/src/components/send-token-page.tsx

```diff
@@ -129,7 +137,7 @@ const useFeeTokenAccountBalance = (
     case StateType.Loaded: {
       return {
         ...accountData,
-        data:
+        amount:
```

Comment:
I actually prefer calling this `data` because that is the shape of the `State` type form `useData`, so before this change this was effectively just a way of mapping (e.g. effectively `map :: T => U => State T => State U`), but now with this change the output of `useFeeTokenAccountBalance` is no longer a `State`.

I'm not sure this matters in practice but I liked the idea that this function is returning a `State<bigint>`, and the fact that it's implemented by mapping over a different `State` isn't important to consumers.